DFC.Digital/DFC.Digital.Service.AzureSearch.UnitTests/DfcSearchQueryServiceTests.cs

```diff
@@ -43,13 +49,47 @@ public async Task SearchAsyncTest()
             await searchService.SearchAsync("searchTerm", dummySearchProperty);
 
             //Assert
-            A.CallTo(() => fakeQueryBuilder.RemoveSpecialCharactersFromTheSearchTerm(A<string>._, A<SearchProperties>._)).MustHaveHappened();
-            A.CallTo(() => fakeQueryBuilder.BuildContainPartialSearch(A<string>._, A<SearchProperties>._)).MustHaveHappened();
-            A.CallTo(() => fakeQueryBuilder.BuildExactMatchSearch(A<string>._, A<string>._, A<SearchProperties>._)).MustHaveHappened();
-            A.CallTo(() => fakeQueryBuilder.TrimCommonWordsAndSuffixes(A<string>._, A<SearchProperties>._)).MustHaveHappened();
+            A.CallTo(() => fakeQueryBuilder.RemoveSpecialCharactersFromTheSearchTerm(A<string>.That.IsEqualTo(SearchTerm), A<SearchProperties>._)).MustHaveHappened();
+            A.CallTo(() => fakeQueryBuilder.TrimCommonWordsAndSuffixes(A<string>.That.IsEqualTo(CleanedSearchTerm), A<SearchProperties>._)).MustHaveHappened();
+            A.CallTo(() => fakeQueryBuilder.BuildContainPartialSearch(A<string>.That.IsEqualTo(TrimmedResults), A<SearchProperties>._)).MustHaveHappened();
+            A.CallTo(() => fakeQueryBuilder.BuildExactMatchSearch(A<string>.That.IsEqualTo(CleanedSearchTerm), A<string>.That.IsEqualTo(PartialTermToSearch), A<SearchProperties>._)).MustHaveHappened();
+            A.CallTo(() => fakeQueryConverter.BuildSearchParameters(A<SearchProperties>._)).MustHaveHappened();
+            A.CallTo(() => fakeIndexClient.Documents).MustHaveHappened();
+            A.CallTo(() => fakeDocumentsOperation.SearchWithHttpMessagesAsync<JobProfileIndex>(A<string>.That.IsEqualTo(BuildExactMatchSearch), A<SearchParameters>._, A<SearchRequestOptions>._, A<Dictionary<string, List<string>>>._, A<CancellationToken>._)).MustHaveHappened();
+            A.CallTo(() => fakeQueryConverter.ConvertToSearchResult(A<DocumentSearchResult<JobProfileIndex>>._, A<SearchProperties>._)).MustHaveHappened();
+        }
+
+        [Theory]
+        [InlineData("*", "/.*.*/ ~")]
```

Comment:
This is wrong isnt it?
we are expecting it to be empty string isnt it?
```suggestion
         [InlineData("*", "")]
```